public/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "MakeSense",
+  "name": "MakeSense.ai Image Annotation",
```

Comment:
makesense.ai - free to use online tool for labelling photos

Reply:
I've edited accordingly.